website/source/docs/builders/amazon-ebs.html.md

```diff
@@ -328,8 +328,15 @@ builder.
     in AWS with the source instance, set the `ssh_keypair_name` field to the name
     of the key pair.
 
--   `ssh_private_ip` (boolean) - If true, then SSH will always use the private
-    IP if available. Also works for WinRM.
+-   `ssh_private_ip` (boolean) - If `true`, then SSH will always use the private
+    IP if available. Also works for WinRM. Overrides `ssh_interface`.
```

Comment:
I wonder if this would be clearer if we make the options mutually exclusive (failing early if both are specified) and frame this change as a deprecation of `ssh_private_ip` in favour of `ssh_interface`? That way the preferred usage is clear.

Reply:
Should I make a docs change tom indicate the deprecation ?

Reply:
yeah I wouldn't have "overrides `ssh_interface`" if they are mutually exclusive